packages/shorebird_cli/lib/src/commands/patch/patch_aar_command.dart

```diff
@@ -152,6 +152,15 @@ of the Android app that is using this module.''',
       releaseVersion: releaseVersion,
     );
 
+    if (release.platformStatuses[ReleasePlatform.android] ==
```

Comment:
Shouldn't we perform this logic as part of a dry run?

Reply:
Makes sense. Updated the dry run early exit here and in the patch android command.